commit 62921ae414c0dc87cc15a1620b50d272a243f6a5
Author: Roman Yurchak <rth.yurchak@pm.me>
Date:   Mon Oct 8 10:53:11 2018 +0200

    Partial fixes for BLAS/LAPACK

diff --git a/scipy/linalg/setup.py b/scipy/linalg/setup.py
index 2c9b9ba22..9a3dae382 100755
--- a/scipy/linalg/setup.py
+++ b/scipy/linalg/setup.py
@@ -14,7 +14,13 @@ def configuration(parent_package='', top_path=None):
 
     config = Configuration('linalg', parent_package, top_path)
 
-    lapack_opt = get_info('lapack_opt')
+    # lapack_opt = get_info('lapack_opt')
+    lapack_opt = {'libraries': ['blas_WA', 'lapack_WA'],
+                  'include_dirs': [],
+                  'library_dirs': ['../../CLAPACK-WA/build/host/'],
+                  'language': 'c',
+                  'define_macros': [('NO_ATLAS_INFO', 1),
+                                    ('HAVE_CBLAS', None)]}
 
     if not lapack_opt:
         raise NotFoundError('no lapack/blas resources found')
diff --git a/scipy/sparse/linalg/eigen/arpack/setup.py b/scipy/sparse/linalg/eigen/arpack/setup.py
index a8175a9d5..26dd537e9 100755
--- a/scipy/sparse/linalg/eigen/arpack/setup.py
+++ b/scipy/sparse/linalg/eigen/arpack/setup.py
@@ -11,7 +11,14 @@ def configuration(parent_package='',top_path=None):
 
     config = Configuration('arpack',parent_package,top_path)
 
-    lapack_opt = get_info('lapack_opt')
+    # lapack_opt = get_info('lapack_opt')
+
+    lapack_opt = {'libraries': ['blas_WA', 'lapack_WA'],
+                  'include_dirs': [],
+                  'library_dirs': ['../../CLAPACK-WA/build/host'],
+                  'language': 'c',
+                  'define_macros': [('NO_ATLAS_INFO', 1),
+                                    ('HAVE_CBLAS', None)]}
 
     if not lapack_opt:
         raise NotFoundError('no lapack/blas resources found')
diff --git a/scipy/sparse/linalg/isolve/setup.py b/scipy/sparse/linalg/isolve/setup.py
index becb9237a..6fa7f1948 100755
--- a/scipy/sparse/linalg/isolve/setup.py
+++ b/scipy/sparse/linalg/isolve/setup.py
@@ -11,7 +11,13 @@ def configuration(parent_package='',top_path=None):
 
     config = Configuration('isolve',parent_package,top_path)
 
-    lapack_opt = get_info('lapack_opt')
+    # lapack_opt = get_info('lapack_opt')
+    lapack_opt = {'libraries': ['blas_WA', 'lapack_WA'],
+                  'include_dirs': [],
+                  'library_dirs': ['../../CLAPACK-WA/build/host/'],
+                  'language': 'c',
+                  'define_macros': [('NO_ATLAS_INFO', 1),
+                                    ('HAVE_CBLAS', None)]}
 
     if not lapack_opt:
         raise NotFoundError('no lapack/blas resources found')
